test/extended/deployments/deployments.go

```diff
@@ -152,7 +152,8 @@ var _ = g.Describe("deploymentconfigs", func() {
 			o.Expect(err).NotTo(o.HaveOccurred())
 
 			g.By(fmt.Sprintf("by checking that the second deployment exists"))
-			err = wait.PollImmediate(500*time.Millisecond, 30*time.Second, func() (bool, error) {
+			// TODO when #11016 gets fixed this can be reverted to 30seconds
+			err = wait.PollImmediate(500*time.Millisecond, 1*time.Minute, func() (bool, error) {
```

Comment:
I've opened https://github.com/openshift/origin/issues/11732 and added this PR there, @kargakis can you add other changes you know of, so it's easier to nail them down faster.